### PR TITLE
Fixing #7264 issue

### PR DESCRIPTION
**Pull request by suraj-webkul**
Issue #7264 fixed

### CODE REVIEW

**Comment by Sanjaybhattwebkul** [comment]
@suraj-webkul 
**Not working**, Still, **duplicate addresses** are getting saved.Please find the **screenshot** [here](https://webkul.chatwhizz.com/share/screenshot/63edd31b93ea2803a944aa7d)

**Comment by Sanjaybhattwebkul** [comment]
@suraj-webkul 
This is **working fine**.

**Review by Sanjaybhattwebkul — DISMISSED** [review]

**Comment by devansh-webkul** [comment]
Conflicts not resolved properly take help from @Sanjaybhattwebkul.

**Review by Sanjaybhattwebkul — APPROVED** [review]

**Comment by Sanjaybhattwebkul** [comment]
@suraj-webkul 
This is working fine

File: packages/Webkul/Velocity/src/Resources/views/shop/checkout/onepage.blade.php [hunk 1]
```diff
@@ -582,6 +579,12 @@ class="theme-btn"
                         this.new_billing_address = true;
                         this.isPlaceOrderEnabled = false;
                         this.address.billing.address_id = null;
+
+                        setTimeout(() => {
+                            if (this.$refs.billingSaveAsAddress && this.$refs.billingSaveAsAddress.checked) {
```
Comment (Sanjaybhattwebkul) [inline packages/Webkul/Velocity/src/Resources/views/shop/checkout/onepage.blade.php]: Break conditions in 2 lines

**Review by Sanjaybhattwebkul — CHANGES_REQUESTED** [review]

File: packages/Webkul/Velocity/src/Resources/views/shop/checkout/onepage.blade.php (hunk 1 above)
Comment (suraj-webkul) [reply packages/Webkul/Velocity/src/Resources/views/shop/checkout/onepage.blade.php]: breaked into 2 lines

**Comment by Amitk-webkul** [comment]
@suraj-webkul 
This is working fine.

**Review by jitendra-webkul — APPROVED** [review]

**Pull request merged**